h/search/index.py

```diff
@@ -26,7 +27,17 @@ class Window(namedtuple('Window', ['start', 'end'])):
     pass
 
 
-def index(es, annotation, request, target_index=None):
+def index(models_annotation, request):
+    search_annotation = persistence.Annotation.create(models_annotation)
+
+    request.registry.notify(AnnotationTransformEvent(
+        request, models_annotation, search_annotation.to_dict(),
+    ))
+
+    search_annotation.save()
+
+
+def index_old(es, annotation, request, target_index=None):
```

Comment:
The parameters that we want for the new `index()` are different than the old one, and `index()` was only called in two places as far as I can see, so I've renamed it to `index_old()` and will change the two places that call it to call both `index_old()` and then `index()` with different args.

Will do the same with `delete()`